packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -45,111 +55,105 @@ const HeaderTools = ({
+        )}
+        <ToolbarGroup
+          alignment={{ default: 'alignRight' }}
+          spacerItems={{ default: 'spacerNone', md: 'spacerMd' }}
         >
-          <GithubIcon />
-        </Button>
-      </PageHeaderToolsItem>
-      {hasVersionSwitcher && (
-        <PageHeaderToolsItem>
-          <Dropdown
-            className="ws-org-version-switcher"
-            onSelect={() => setDropdownOpen(!isDropdownOpen)}
-            toggle={(
-              <DropdownToggle
-                className={`ws-org-version-toggle${isDropdownOpen ? '-expanded': ''}`}
-                onToggle={() => setDropdownOpen(!isDropdownOpen)}
-              >
-                Release {initialVersion.name}
-              </DropdownToggle>
-            )}
-            isOpen={isDropdownOpen}
-            dropdownItems={[
-              <DropdownGroup key="latest" label="Latest">
-                {getDropdownItem(latestVersion)}
-              </DropdownGroup>,
-              <DropdownGroup key="Previous" label="Previous releases">
-                {Object.values(versions.Releases)
-                  .filter(version => !version.hidden && !version.latest)
-                  .slice(0,3)
-                  .map(getDropdownItem)}
-              </DropdownGroup>,
-              <Divider key="divider" className="ws-switcher-divider"/>,
-              <DropdownItem
-                key="PatternFly 3"
-                className="ws-patternfly-3"
-                target="_blank"
-                href="https://pf3.patternfly.org/"
-              >
-                PatternFly 3
-                <ExternalLinkAltIcon />
-              </DropdownItem>
-            ]}
-          />
-        </PageHeaderToolsItem>
-      )}
-    </PageHeaderTools>
+          {hasDarkThemeSwitcher && (
+            <ToolbarItem>
+              <Switch id="ws-theme-switch" label="Dark theme" defaultChecked={false} onChange={() =>
+                document.querySelector('html').classList.toggle('pf-theme-dark')} />
+            </ToolbarItem>
+          )}
+          {hasSearch && (
+            <ToolbarItem>
+              <SearchInput
+                className="ws-global-search"
+                placeholder="Search"
+                value={searchValue}
+                onChange={onChange}
+                onClear={() => onChange('')}
+                expandableInput={{ isExpanded: isSearchExpanded, onToggleExpand, toggleAriaLabel: 'Expandable search input toggle' }}
+              />
+            </ToolbarItem>
+          )}
+          <ToolbarItem>
+            <Button
+              component="a"
+              variant="link"
+              href="//github.com/patternfly"
+              target="top"
+              aria-label="Link to PatternFly GitHub page"
+              className="ws-github-pageheader"
+            >
+              <GithubIcon />
+            </Button>
+          </ToolbarItem>
+          {hasVersionSwitcher && (
+            <ToolbarItem>
+              <Dropdown
```

Comment:
I believe this should be the full height variation, that's what we're doing with dropdowns in the masthead like this AFAIK. Also there are custom styles on this making it shift around on hover and when it's expanded - IMO it would be much better as a plain ol' dropdown without the custom stuff. 

![Nov-17-2022 18-59-29](https://user-images.githubusercontent.com/35148959/202592359-47912f4b-b916-49ca-b58f-0242487ac8dc.gif)

Reply:
@mcarrano @mceledonia wdyt? ^